pvt/helpers/src/contract.ts

```diff
@@ -30,7 +28,7 @@ export async function deploy(
 
   const artifact = getArtifact(contract);
 
-  const factory = await ethers.getContractFactoryFromArtifact(artifact, { signer: from, libraries });
+  const factory = await ethers.getContractFactory(artifact.abi, artifact.bytecode, from);
```

Comment:
I think we also would need to send the libraries to link here: https://v2.hardhat.org/hardhat-runner/plugins/nomicfoundation-hardhat-ethers